integration-tests/.storybook/config.js

```diff
@@ -38,4 +38,5 @@ setOptions({
   showSearchBox: false,
   downPanelInRight: true,
   sortStoriesByKind: false,
+  hierarchySeparator: /\./,
```

Comment:
This is a new feature in `3.2`, which potentially touches the story groups - currently only in UI, but I think it is important we make it part of the test suite for the future.